packages/gatsby-theme-aio/src/components/Tabs/index.js

```diff
@@ -108,11 +109,12 @@ const Tabs = forwardRef(({
   }, [onFontsReady]);
 
   return (
+
     <div
       ref={ref}
       {...props}
-      role='tablist'
-      aria-orientation={orientation}
+      role={!isHeader ? 'tablist' : undefined}
```

Comment:
Maybe I don't understand the isHeader condition but does it need to modify the role and orientation?